apps/vscode/src/appdev/devServerGuard.cjs

```
@@ -0,0 +1,139 @@
+ *      stdout/stderr, so the extension's banner/build parsing sees
+ *      rsbuild verbatim.
+ *   3. Mirror the exit — the guard exits with the child's code, so the
+ *      extension's crash detection treats the guard as the server.
+ *
+ * argv: [ownerPid, useShell(0|1), cmd, ...args]
+ * cwd:  the app folder (set by the spawner).
+ */
+
+'use strict';
+
+const { spawn } = require('child_process');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)